### ISSUE TITLE
[Bug] Incorrect `${ref}` variable in the custom command

### ISSUE THREAD

**Issue by 3mini** [ForkIssues/Tracker#2069]
## Environment

- macOS Sonoma 14.3.1
- Fork Version 2.40

## Description

I made a custom command as below.

<details><summary>custom-commands.json</summary>

```json
[
  {
    "name" : "Custom Command",
    "target" : "repository",
    "ui" : {
      "buttons" : [
        {
          "action" : {
            "script" : "echo $1{sha}\necho $1{sha:abbr}\necho $1{ref}\necho $1{ref:short}\necho $1{ref:full}",
            "showOutput" : true,
            "type" : "sh",
            "waitForExit" : true
          },
          "title" : "OK"
        },
        {
          "action" : {
            "type" : "cancel"
          },
          "title" : "Cancel"
        }
      ],
      "controls" : [
        {
          "dropdownType" : "references",
          "filter" : "refs\/tags\/",
          "title" : "Tag",
          "type" : "dropdown"
        }
      ],
      "description" : "",
      "title" : "Tag"
    }
  }
]
```

</details>

As shown in the screenshots, the **selected tag name** was expected to be used as the `$1{ref}` variable in the `OK` action, but it references a different value.

<img width="1100" alt="{ref}" src="https://github.com/fork-dev/Tracker/assets/9378144/f5570b7c-0605-4a64-b47f-27f18637b645">

<img width="1264" alt="image" src="https://github.com/fork-dev/Tracker/assets/9378144/d0567995-8ee3-4a4e-9e97-7fcab5ba7460">

<img width="1264" alt="image" src="https://github.com/fork-dev/Tracker/assets/9378144/921abdca-b94d-4ee0-8e30-916aa217b716">

**Comment by DanPristupov** [comment]
Thank you for reporting this. I've reproduced the problem.

**Comment by DanPristupov** [comment]
We just released Fork 2.40.2 with the fix. Please check for updates.

**Comment by 3mini** [comment]
I've confirmed that the issue has been resolved. Thanks for your quick response.
